Makefile

```diff
@@ -54,7 +49,7 @@ GOBIN=$(shell go env GOBIN)
 endif
 
 # Make sure GOBIN is on the PATH
-export PATH := $(GOBIN):$(PATH)
+export PATH := ${BIN}:$(GOBIN):$(PATH)
```

Comment:
Temporarily added so that I can use a docker-cli in `bin/` rather than my system installed podman, which is too slow:
 * https://github.com/containers/buildah/issues/1714